web_programming/web_scrapping.py

```diff
@@ -0,0 +1,144 @@
+import csv
+import re
+
+import bs4
+import requests
+
+
+def scrap_info(request_url: str) -> None:
```

Comment:
This looks pretty cool!!

According to `CONTRIBUTING.md` algorithmic functions should take in inputs, return outputs, and NOT print.

Instead of returning `None`, please return `meta_info` (perhaps with a more self-documenting name like `webpage_metadata`).  This will allow the caller to determine if they want to store this data as a csv (put that code in a separate function) or Tweet some key attribute(s) or put it into Pandas or use it in a GitHub Action.

Reply:
Done in the latest commit